api/nakadi-event-bus-api.yaml

```diff
@@ -799,6 +799,112 @@ paths:
             $ref: '#/definitions/Problem'
 
 
+  /subscriptions/{subscription_id}/cursors:
+    get:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: Exposes the currently committed offsets of a subscription.
+      parameters:
+        - name: subscription_id
+          in: path
+          type: string
+          description: Id of subscription
+          required: true
+      responses:
+        '200':
+          description: Ok
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Cursor'
```

Comment:
Good point. With current situation user will have to do additional request to know the event-type of this subscription, but it is not really a show stopper.
In general I see no problem to add event-type to cursor, because it is backwards compatible change. But I think the right moment to do that is when we switch to multi-event-type consumption (for low and high levels)